express-get-json/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000);
+
+app.use('/', (req, res) => {
+  if (req.path === '/api/grades') {
+    res.json(grades);
+  } else {
```

Comment:
- use `app.get()` instead of `app.use()`
- the client should receive an array and not an object